test/unit/menulst.test.js

```diff
@@ -387,27 +387,24 @@ tap.test('Linux CN', function linuxCN(t) {
     };
     const plat = fnParams.bootParams.platform;
     menuLst.buildMenuLst(fnParams, function (menu) {
-        t.deepEqual(menu.split('\n'), MENU_START.concat([
-            'variable os_console console=ttyS0',
+            /* BEGIN JSSTYLED */
+            /* eslint-disable max-len */
+            t.deepEqual(menu.split('\n'), MENU_START.concat([
+            'variable os_console console=ttyS1,115200n8',
             '',
             'title Live 64-bit',
-            format('   kernel$ /os/%s/platform/x86_64/vmlinuz', plat),
+            format('   kernel$ /os/%s/platform/x86_64/vmlinuz console=tty0 console=ttyS1,115200n8', plat),
```

Comment:
"make check" is not gonna like this, IIRC.

Reply:
I thought that initially too, but it passes.

I think it's because it's an argument to `format()` and it's a single quoted string, so it treats it as if it's a single token?